wagtailautocomplete/widgets.py

```diff
@@ -2,9 +2,15 @@
 
 from django.apps import apps
 from django.forms import Widget
+from wagtail import VERSION
 
 from .views import render_page
 
+if VERSION > (2.0):
```

Comment:
Apologies—my suggestion here was wrong. This should be `VERSION > (2, 0)` (where `(2, 0)` is a tuple, not a float `(2.0)`. I'll make this change and if that seems to be the only issue, I'll rebase and merge myself.